contributingGuides/PERFORMANCE.md

```diff
@@ -1,52 +1,275 @@
 - Watch out for [very large lists](https://reactnative.dev/docs/optimizing-flatlist-configuration) and things like `Image` components re-fetching images on render when a remote uri did not change.
-- Avoid the temptation to over-optimize. There is added cost in both code complexity and performance when adding checks like `shouldComponentUpdate()`. Be selective about when you use this and make sure there is a measurable difference before proposing the change. As a very general rule, it should be measurably faster to run logic to avoid the re-render (e.g. do a deep comparison) than it would be to let React take care of it without any extra intervention from us.
+- Avoid the temptation to over-optimize. There is added cost in both code complexity and performance when using memoization hooks like `useMemo` and `useCallback`. Be selective about when you use them and make sure there is a measurable difference before proposing the change. As a very general rule, it should be measurably faster to run the memoized logic than it would be to let React re-render the component without any extra intervention from us.
 - Use caution when adding subscriptions that might re-render very large trees of components e.g. subscribing to state that changes often (current report, current route, etc) in the app root.
-- Avoid using arrow function callbacks in components that are expensive to re-render. React will re-render this component since each time the parent renders it creates a new instance of that function. **Alternative:** Bind the method in the constructor instead.
+- Avoid passing new functions as props to components on every render. This can be avoided by using `useCallback` or by defining the function outside of the component.
 
 ## Tools
 
-### Chrome Dev Tools > Performance > Timing (Web Only)
+### **Web**: Chrome Dev Tools > Performance
 
 - Profiling in Chrome Dev Tools performance tab in the "Timing" section
 - This will show various components and how long they took to render. It can be a little intense to dig through it all at first, but the more time you spend with it the easier it gets to separate the signal from noise.
-- The timing information might be inaccurate in development mode since this slows things down a ton. However, it's still useful for seeing which things take the longest and it's not too difficult to look and see which things are re-rendering.
+- The timing information might be inaccurate in development mode since this slows things down a ton. However, it's still useful for seeing which things are re-rendering. You can also use the React DevTools Profiler to get more accurate timing information.
+
+#### Steps to Profile:
+
+1. **Open Chrome DevTools (`cmd+shift+j`)**
```

Comment:
I think its option on Max
```suggestion
1. **Open Chrome DevTools (`cmd+option+j`)**
```

Reply:
you're right, thanks for spotting this 🥇